wiki/Tournaments/MUKASHI/1/en.md

```diff
@@ -0,0 +1,214 @@
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2022-11-14/2022-11-27 |
+| Technical meeting | 2022-11-30 |
+| Opening | 2022-12-03 (11:00 UTC +7) |
+| Extra Preliminary Match | 2022-12-03 (11:15 UTC +7) |
+| Quarterfinal 1 | 2022-12-03 (10:50 UTC +7) |
+| Quarterfinal 2 | 2022-12-03 (12:25 UTC +7) |
+| Quarterfinal 3 | 2022-12-03 (13:00 UTC +7) |
+| Quarterfinal 4 | 2022-12-03 (13:35 UTC +7) |
+| *Break* | 2022-12-03 (14:00 UTC +7) |
+| Semifinal 1 | 2022-12-03 (14:30 UTC +7) |
+| Semifinal 2 | 2022-12-03 (15:05 UTC +7) |
+| 3rd Place Match | 2022-12-03 (15:40 UTC +7) |
+| Grand Final | 2022-12-03 (16:30 UTC +7) |
+| Closing and awarding | 2022-12-03 (17:00 UTC +7) |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 500,000 IDR, commemorative certificate |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 300,000 IDR, commemorative certificate |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 200,000 IDR, commemorative certificate |
+
+## Organisation
+
+The osu!mania 4K Community Tournament @ MUKASHI FESTIVAL ID Senayan Park was run by various POST representatives and Indonesian osu! community members.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Host | ::{ flag=ID }:: Raben (POST), ::{ flag=ID }:: Ijul (POST), ::{ flag=ID }:: Erlangga (POST) |
+| Steering committee | ::{ flag=ID }:: [Niva](https://osu.ppy.sh/users/197805), ::{ flag=ID }:: [Vvn](https://osu.ppy.sh/users/3204677) |
+| Mappool selector | ::{ flag=ID }:: [Mipha-](https://osu.ppy.sh/users/5767941), ::{ flag=ID }:: [Symmatrix-](https://osu.ppy.sh/users/17258072), ::{ flag=ID }:: [IceVee](https://osu.ppy.sh/users/8108213) |
+| Mappool helper | ::{ flag=ID }:: [Vvn](https://osu.ppy.sh/users/3204677), ::{ flag=ID }:: [Sies](https://osu.ppy.sh/users/6491991) |
+| Commentator | ::{ flag=ID }:: [Niva](https://osu.ppy.sh/users/197805), ::{ flag=ID }:: [Vvn](https://osu.ppy.sh/users/3204677), ::{ flag=ID }:: [Maxus](https://osu.ppy.sh/users/4335785) |
+| Referee | ::{ flag=ID }:: [Niva](https://osu.ppy.sh/users/197805) |
+| Documentation | ::{ flag=ID }:: [Vvn](https://osu.ppy.sh/users/3204677), ::{ flag=ID }:: [liykun](https://osu.ppy.sh/users/9500057) |
+| Wiki editor | ::{ flag=ID }:: [Niva](https://osu.ppy.sh/users/197805) |
+| Additional helper | ::{ flag=ID }:: [Ascaveth](https://osu.ppy.sh/users/3245206), ::{ flag=ID }:: [Suikami](https://osu.ppy.sh/users/1929336) |
+
+## Links
+
+- **[Technical meeting document](https://docs.google.com/presentation/d/14yBMl4KiTAYxBxLrECJgixUlQfMkcKrQsYNi5wx-oOs/edit#slide=id.p)**
+- [POST website](https://santai.web.id/)
```

Comment:
website says "POS" and not "POST"?

Reply:
yea to my knowledge there are instances where they went with `POS` `POST` and even `POS Total`

i think i'll keep everything here in the article as `POST` for consistency tho